src/app/shared/components/Field/PlayerCard/index.tsx

```diff
@@ -40,7 +40,7 @@ const PlayerCard = (props: PlayerCardProps) => {
   return (
     <div className={className}>
       <div
-        className="box-border cursor-pointer h-15 md:h-20 w-15 md:w-20 border-2 hover:border-cyan-300"
```

Comment:
turns out `h-15` is not a value so this was doing nothing

https://tailwindcss.com/docs/height